pages/ugm2024.md

```diff
@@ -22,11 +22,11 @@ Keynote speakers will be announced in Spring 2024.
 The agenda will be announced in Spring 2024.
 
 ## Abstracts and Registration
-*Abstract and bursary application deadline: Friday, March 15, 2024*
+*Abstract and bursary application deadline: Monday, April 1, 2024*
```

Comment:
Maybe extend by a month to April 15? Do we need to know the numbers that early?

Reply:
We had hoped to put together a draft program before the registration opens on April 1st, so that people had more information available before they signed up to attend. However, I think we'll have enough information to give an idea of the sessions that we can organize (e.g., drug discovery, soft mater, developing toolkits, etc.) without listing specific names/talk titles.

I'd support the April 15 deadline. @micaela-matta and @RMeli - I know you both approved the April 1 deadline on Discord; thoughts on April 15 instead?

Reply:
I _might_ look at a list of invited speakers for a conference but I _never_ look at the full program before deciding to go somewhere. With keynote speakers and general sessions planned out, you should have enough information.

Reply:
I'm happy with a longer extension too, whatever you guys thing it's best. I'd agree with @orbeckst that keynote speakers + conference topics is usually enough to choose if it's worth going to a conference.